crates/amaru-ledger/src/store/columns/dreps.rs

```diff
@@ -12,22 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use amaru_kernel::{cbor, Anchor, Epoch, Lovelace, StakeCredential};
+use amaru_kernel::{cbor, Anchor, Epoch, Lovelace, Slot, StakeCredential};
 use iter_borrow::IterBorrow;
 
 pub const EVENT_TARGET: &str = "amaru::ledger::store::dreps";
 
 /// Iterator used to browse rows from the Accounts column. Meant to be referenced using qualified imports.
 pub type Iter<'a, 'b> = IterBorrow<'a, 'b, Key, Option<Row>>;
 
-pub type Value = (Option<Anchor>, Option<Lovelace>, Epoch);
+pub type Value = (Option<Anchor>, Option<Lovelace>, Slot, Epoch);
 
 pub type Key = StakeCredential;
 
 #[derive(Debug, Clone, PartialEq, Default)]
 pub struct Row {
     pub deposit: Lovelace,
     pub anchor: Option<Anchor>,
+    pub registered_at: Slot,
```

Comment:
We're going to need three indices here in the end: Slot, transaction position within the block, and certification position within the transaction. The ordering must be total.

Reply:
Since we these indices will carry a quite precise semantic, then a dedicated type is indicated (e.g. `CertificatePointer`, where we define and test the ordering behaviour accordingly).